src/content/validated-patterns/index.json

```
@@ -0,0 +1,12 @@
+{
+	"sidebarCategories": [],
+	"landingPage": {
+		"hero": {
+			"heading": "Validated Patterns",
+		},
+		"overview": {
+			"heading": "What are Validated Patterns?",
+			"body": "HashiCorp Validated Patterns provide prescriptive guidance on integrating HashiCorp and partner technologies.",
```

The `,` needs to be removed to fix this error `Error: Cannot parse JSON: Unexpected token "}"`
```suggestion
			"heading": "Validated Patterns"
		},
		"overview": {
			"heading": "What are Validated Patterns?",
			"body": "HashiCorp Validated Patterns provide prescriptive guidance on integrating HashiCorp and partner technologies."
```